akka-projection-grpc/src/main/scala/akka/projection/grpc/replication/javadsl/Replication.scala

```diff
@@ -28,6 +28,9 @@ import akka.projection.grpc.producer.javadsl.EventProducer
 import akka.projection.grpc.producer.javadsl.EventProducerSource
 import akka.projection.grpc.replication.internal.ReplicationImpl
 
+import java.util.Optional
+import scala.jdk.OptionConverters.RichOption
```

Comment:
Not important, but I'd prefer that we stick to `import scala.compat.java8.OptionConverters._` until we decide to switch all over the place.